assets/semgrep_rules/client/cryptography-random-usage.yaml

```diff
@@ -0,0 +1,36 @@
+rules:
```

Comment:
it seems this rule is doubling up with other rules (all the ones I'm tagged in)

Could we remove the other rules if we're going to independently manage them here (I prefer this separate rule for this)